openapi/paths/transactions@{id}@query.yaml

```
@@ -0,0 +1,48 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+  - $ref: ../components/parameters/organizationId.yaml
+post:
+  tags:
+    - Transactions
+  summary: Query a Transaction status
+  operationId: PostTransactionQuery
+  description: Query a Transaction status from payment gateway.
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: ../components/schemas/Transactions/TransactionQuery.yaml
+    required: true
+  responses:
+    '200':
+      description: Transaction query was done successfully.
+      headers:
+        Rate-Limit-Limit:
+          $ref: ../components/headers/Rate-Limit-Limit.yaml
+        Rate-Limit-Remaining:
+          $ref: ../components/headers/Rate-Limit-Remaining.yaml
+        Rate-Limit-Reset:
+          $ref: ../components/headers/Rate-Limit-Reset.yaml
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/Transactions/Transaction.yaml
+    '201':
+      description: Transaction query was done successfully and parsed.
```

```suggestion
      description: Transaction query was done successfully and the transaction was reconciled.
```
Do you think such description may be appropriate?